turbo/debug/flags.go

```diff
@@ -266,7 +266,8 @@ func Setup(ctx *cli.Context, rootLogger bool) (log.Logger, *tracers.Tracer, *htt
 func StartPProf(address string, metricsMux *http.ServeMux) *http.ServeMux {
 	cpuMsg := fmt.Sprintf("go tool pprof -lines -http=: http://%s/%s", address, "debug/pprof/profile?seconds=20")
 	heapMsg := fmt.Sprintf("go tool pprof -lines -http=: http://%s/%s", address, "debug/pprof/heap")
-	log.Info("Starting pprof server", "cpu", cpuMsg, "heap", heapMsg)
+	torrentMsg := fmt.Sprintf("curl -s http://%s/downloader/torrentClientStatus > torrentStatus.txt", address)
```

Comment:
Problem is in next code: `debugMux := cmp.Or(metricsMux, pprofMux)`. now it added to 1 of 2 ports (I guess must be added to both).

Reply:
since we'll specify only one address to the user, it's enough to add to just one.

Reply:
I think now `--pprof` doesn't enable metrics and `--metrics` doesn't enable proof. and they are on 2 different ports (don't remember if Geth does the same or not)

Reply:
ah yes; we need to chose based on what is enabled. That now happens in the PR.